receiver/skywalkingreceiver/README.md

```diff
@@ -0,0 +1,24 @@
+# Skywalking Receiver
+
+Receives trace data in [Skywalking](https://skywalking.apache.org/) format.
+
+Supported pipeline types: traces
```

Comment:
We might want to add the status of the receiver or at least mention it's not yet fully functional